core/domain/user_services.py

```diff
@@ -687,6 +687,21 @@ def create_new_user(user_id, email):
     return user_settings
 
 
+def get_system_username(system_user_id):
```

Comment:
I don't think this works. What if a bad user id, that doesn't meet either condition, is passed in?

Here's what I suggest instead: have a top-level constant dict mapping system IDs to system usernames. Then, at the start of each function, do something like:

    if user_id in DICT:
        return DICT[user_id]

(pick a better name for DICT :) )

Reply:
Yup right, one question - should I do that in feconf.py or in this file only?

Reply:
If this dict is needed only in this file, then do it in this file; if it's needed in multiple files, do it in feconf.py.

Reply:
Done.